src/main/kotlin/no/nav/familie/pdf/pdf/UtilsMetaData.kt

```diff
@@ -0,0 +1,55 @@
+package no.nav.familie.pdf.pdf
+
+import com.itextpdf.kernel.pdf.PdfName
+import com.itextpdf.kernel.pdf.PdfString
+import com.itextpdf.kernel.pdf.PdfViewerPreferences
+import com.itextpdf.kernel.xmp.XMPMeta
+import com.itextpdf.kernel.xmp.XMPMetaFactory
+import com.itextpdf.pdfa.PdfADocument
+
+/**
+ * `UtilsMetaData` er en hjelpekklasse for å legge til metadata i et PDF-dokument.
+ * Vi setter metadataene på to ulike måter for å dekke forskjellige behov:
+ * 1. `documentInfo`: Grunnleggende metadata (tittel, skaper) som brukes av PDF-lesere for visning i dokumentegenskaper.
+ * 2. `setXmpMetadata`: Avansert, maskinlesbar XMP-metadata for standarder som PDF/UA
+ */
+object UtilsMetaData {
```

Comment:
Burde dette være en klasse?

Reply:
nei, 

"den inneholder funksjoner og trenger ikke å opprettholde noen tilstand mellom kall, er det hensiktsmessig å definere det som et objekt. Dette gir en enkel tilgang til funksjonene uten behov for å instansiere klassen."